config/data_centers/data_centers.json

```diff
@@ -679,6 +679,36 @@
     "source": "https://docs.aws.amazon.com/global-infrastructure/latest/regions/aws-regions.html",
     "operationalSince": "2021"
   },
+  "hlrs-de-stuttgart": {
+    "provider": "hlrs",
+    "displayName": "HLRS Stuttgart",
+    "zoneKey": "DE-BW",
+    "lonlat": [9.0970622, 48.739457],
+    "region": "de-stuttgart",
+    "source": "https://www.hlrs.de/",
+    "operationalSince": null,
+    "status": "operational"
+  },
+  "data-center-stuttgart-vaihingen-1": {
+    "provider": "itenos",
+    "displayName": "Itenos data center Stuttgart-Vaihingen-1",
+    "zoneKey": "DE-BW",
+    "lonlat": [9.1285794, 48.7214015],
+    "region": "de-stuttgart",
```

Comment:
This region should be the region that the data center owner uses.
So `stuttgart-vaihingen-1` from what I can tell.

Possibly just `stuttgart-vaihingen` if there are more than one unit to the same datacenter region located at the same location.